api/urls.py

```diff
@@ -16,9 +16,10 @@
     path('company/', CompanyListAPIView.as_view(), name='company'),
 
     re_path('^employee/(?P<id>\d+)$', PersonOneAPIView.as_view(), name='one_employee'),
-    re_path('^employees_by_company_id/(?P<id>\d+)$', PersonsByCompanyIdAPIView.as_view(), name='one_employee_by_company_id'),
-    path('employee/', PersonListAPIView.as_view(), name='employees'),
+    re_path('^employees_by_company_id/(?P<id>\d+)$', PersonsByCompanyIdAPIView.as_view(),
```

Comment:
RESTfully this should like `GET /company/:id/employee` OR `GET /employee?company_id=...`